CONTRIBUTING.md

```diff
@@ -64,6 +64,11 @@ You can run up your changes locally to test them in a few different ways.
   
   (see `alternativeIdePath` in the Gradle IntelliJ Plugin [documentation](https://github.com/JetBrains/gradle-intellij-plugin) for more details)
 
+- If you wish to run the integration tests, they require valid AWS credentials to run. Take care that it will use your default AWS profile if it exists.
```

Comment:
```suggestion
- If you wish to run the integration tests, they require valid AWS credentials to run. Take care, as it will respect any credentials currently defined in your environmental variables, and fallback to your default AWS profile otherwise.
```